terraform/examples/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Ignore examples' state
```

Comment:
We already ignore it in main `.gitignore`

Reply:
Updated.